src/software/simulation/er_force_simulator.cpp

```diff
@@ -137,6 +137,14 @@ void ErForceSimulator::setWorldState(const TbotsProto::WorldState& world_state)
     {
         setRobots(world_state.yellow_robots(), gameController::Team::YELLOW);
     }
+
+    auto sim_started_trigger_msg = *createSimulationStartedTrigger(true);
+    if (!has_sent_sim_start_trigger)
+    {
+        LOG(VISUALIZE) << sim_started_trigger_msg;
+        has_sent_sim_start_trigger = true;
+        std::cout << sim_started_trigger_msg.DebugString() << std::endl;
+    }
```

Comment:
We should probably move this to the end of `ErForceSimulator` constructor. Then shouldn't need `has_sent_sim_start_trigger` either

Reply:
Actually on a second thought, this might still cause a race condition, since the `ErForceSimulator` object is instantiated (and the constructor called) before we add the WorldState listener. 
https://github.com/UBC-Thunderbots/Software/blob/0ad50029569376bf4f62ca5918bad95e47ca73a6/src/software/er_force_simulator_main.cpp#L75

https://github.com/UBC-Thunderbots/Software/blob/783fcf0ef3d72f6713914934df10e8f3ac80a760/src/software/er_force_simulator_main.cpp#L125-L130

But we want the `SimulatedStartedTrigger` to be sent after `ErForceSimulator` is ready AND the listener is set up for WorldState to make sure that the WorldState messages aren't missed.

So the better approach would be to send the `SimulatedStartedTrigger` proto after everything is set up in `er_force_simulator_main.cpp`. 
It's also a bit weird to use `LOG(VISUALIZE)` to send a protobuf that is not for visualization. So to keep with how we've done it else where (mainly in `er_force_simulator_main.cpp`), we should create a `ThreadedProtoUnixListener<TbotsProto::SimulatedStartedTrigger>` and `sendProto` using that. 
E.g. of similar ones:
https://github.com/UBC-Thunderbots/Software/blob/783fcf0ef3d72f6713914934df10e8f3ac80a760/src/software/er_force_simulator_main.cpp#L121-L122

https://github.com/UBC-Thunderbots/Software/blob/783fcf0ef3d72f6713914934df10e8f3ac80a760/src/software/er_force_simulator_main.cpp#L189